Modules/IO/MeshBase/ITKKWStyleOverwrite.txt

```diff
@@ -0,0 +1 @@
+itkMeshFileReader\.h Comments Disable
```

Comment:
Thank you Matt. But why is KWStyle check disabled for "itkMeshFileReader.h"?

Reply:
Hmm, possibly this was required in a previous revision -- I will remove it.